include/openlibm_math.h

```diff
@@ -26,7 +26,7 @@
 #endif
 
 #ifndef __arm__
-#define LONG_DOUBLE
+#define OPENLIBM_LONG_DOUBLE
```

Comment:
Can we make this `OLM_LONG_DOUBLE`? We use the `OLM_` prefix in other places so it would be nice to be consistent.

Reply:
Thanks, I will do that